deploy/test/test_cases/TEST_ID_22_helm_rbac_defaults_taken_successfully.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -euxo pipefail
+
+# This test verifies that if the user does not override Helm defaults, Helm will name K8s resources by their defaults and
+# the Secrets Provider will deploy and complete successfully
+# TODO verify Job start up and completed logs
+
+setup_helm_environment
+
+pushd ../../../
+  fill-helm-chart-no-override-defaults
+  helm install -f "secrets-provider/ci/take-defaults-test-values.yaml" secrets-provider ./secrets-provider --set-file environment.conjur.sslCertificate.value="deploy/test/test_cases/conjur.pem"
```

Comment:
can we break this line throughout the repo? 
also - if it repeats itself consider to extract it

Reply:
> if it repeats itself consider to extract it

so this is not a 💯 correct. We have variations where the conjur.pem path is different, where the chart name is different, and where the `values.yml` is different depending on the test so I purposely didn't wrap this in another function because I wanted it to be clear what is happening and which one I am taking for the install

> can we break this line throughout the repo?

sure thing!